connector/web3/web3.go

```diff
@@ -3,31 +3,36 @@ package web3
 
 import (
 	"fmt"
-
+	"github.com/dexidp/dex/connector"
+	lg "github.com/dexidp/dex/pkg/log"
```

Comment:
Let's not do this rename. It's `log` in the rest of the codebase, it'll be alright!